networking/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,16 +13,63 @@ These release notes track the development of the Network Observability Operator
+
+[id="must-gather-1.3"]
+==== Troubleshooting with the must-gather tool
+* Information about the Network Observability Operator can now be included in the must-gather data for troubleshooting. For more information, see xref:../../networking/network_observability/troubleshooting-network-observability.adoc#network-observability-must-gather[Network Observability must-gather].
+
+[id="multi-arch-1.3"]
+==== Multiple architectures now supported 
+* Network Observability Operator can now run on an amd64, ppc64le, or arm64 architecture. Previously, it only ran on amd64. 
+
+[id="deprecated-features-1.3"]
+=== Deprecated features
+
+[id="authToken-host"]
+==== Deprecated configuration parameter setting
+The release of Network Observability Operator 1.3 deprecates the `spec.Loki.authToken` `HOST` setting. You must now only use the `FORWARD` or `DISABLED` settings.
+
+[id="network-observability-operator-1.3.0-bug-fixes"]
+=== Bug fixes
+* Previously, when the Operator was installed from the CLI, the `Role` and `RoleBinding` that are necessary for the Cluster Monitoring Operator to read the metrics were not installed as expected. The issue did not occur when the operator was installed from the web console. Now, either way of installing the Operator installs the required `Role` and `RoleBinding`. (link:https://issues.redhat.com/browse/NETOBSERV-1003[*NETOBSERV-1003*])
+
+* Since version 1.2, the Network Observability Operator can raise alerts when a problem occurs with the flows collection. Previously, due to a bug, the related configuration to disable alerts, `spec.processor.metrics.disableAlerts` was not working as expected and sometimes ineffectual. Now, this configuration is fixed so that it is possible to disable the alerts. (link:https://issues.redhat.com/browse/NETOBSERV-976[*NETOBSERV-976*])
+
+* Previously, when Network Observability was configured with `spec.loki.authToken` set to `DISABLED`, only a `kubeadmin` cluster administrator was able to view network flows. Other types of cluster administrators received authorization failure. Now, any cluster administrator is able to view network flows. (link:https://issues.redhat.com/browse/NETOBSERV-972[*NETOBSERV-972*])
+
+* Previously, a bug prevented users from setting `spec.consolePlugin.portNaming.enable` to `false`. Now, this setting can be set to `false` to disable port-to-service name translation. (link:https://issues.redhat.com/browse/NETOBSERV-971[*NETOBSERV-971*])
+
+* Previously, the metrics exposed by the console plugin were not collected by the Cluster Monitoring Operator (Prometheus), due to an incorrect configuration. Now the configuration has been fixed so that the console plugin metrics are correctly collected and accessible from the {product-title} web console. (link:https://issues.redhat.com/browse/NETOBSERV-765[*NETOBSERV-765*])
+
```

Comment:
@skrthomas NETOBSERV-1070 affects 1.2 release as well, we should probably mention under Bug fix. Something along the lines: Previously, when `processor.metrics.tls` is set to AUTO in the `FlowCollector`, the `flowlogs-pipeline` `servicemonitor` is not adapted to the appropriate TLS scheme and metrics were not be visible under OCP console. Now the issue fixed for AUTO mode.

Reply:
tysm Mehul!  I think I misunderstood when we were talking about NETOBSERV-1070 previously, I guess I was thinking that only NETOBSERV-1087 was a known issue and I removed 1070. I will add 1070 back to the Bug Fix list.

Reply:
I think I previously had it as a Known Issue..